### PR TITLE
[#1657] VaRadio readonly state, VaRadio & VaSwitch examples updated

### PR DESCRIPTION
**Pull request by aluarius**
close: #1657 

## Description
- `VaSwitch` has readonly state. Example added.
- Added readonly state for `VaRadio` (and example). I'd thought about implementing this via `useSelectable`, but it is not suitable in its current form, and its adaptation looks like an unnecessary complication of the common composable cause of a single case.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

**Comment by rustem-nasyrov** [comment]
What's the difference here between `readonly` and `disabled` states for the `VaRadio`?

**Comment by aluarius** [comment]
> What's the difference here between `readonly` and `disabled` states for the `VaRadio`?

Difference between `readonly` and `disabled`  everywhere is only in appearance - readonly has a usual components view, disabled - the same, but with opacity.

**Comment by rustem-nasyrov** [comment]
> > What's the difference here between `readonly` and `disabled` states for the `VaRadio`?
> 
> Difference between `readonly` and `disabled` everywhere is only in appearance - readonly has a usual components view (without ability to interact), disabled - the same, but with opacity.
> 
>  

Thank you, now I got it. :)

**Review by LighthouseKeeperYN — APPROVED** [review]

**Review by rustem-nasyrov — CHANGES_REQUESTED** [review]
For `readonly` state you need to add a couple of CSS code:
```
cursor: default;
pointer-events: none;
```

**Review by RVitaly1978 — CHANGES_REQUESTED** [review]
I think it would be better if we add this CSS for the `readonly` state of the components:
```
cursor: default;

.component-label-class-selector {
  cursor: initial;
}
```
in this case, it will be clearer for the user that the text of the label can be selected.

**Review by RVitaly1978 — APPROVED** [review]

File: packages/ui/src/components/va-switch/VaSwitch.vue [hunk 1]
```diff
@@ -251,6 +252,15 @@ export default defineComponent({
     @include va-disabled;
   }
 
+  &--readonly {
+    @include va-readonly;
+
+    .va-switch__label {
+      cursor: initial;
+      pointer-events: auto;
+    }
```
Comment (rustem-nasyrov) [inline packages/ui/src/components/va-switch/VaSwitch.vue:261]: Doesn't it inherits from parent `--readonly` class?

Comment (aluarius) [reply packages/ui/src/components/va-switch/VaSwitch.vue:261]: No. Everything was tested and works as Vitaly suggested - "arrow" cursor for interactive area and "text" cursor for label.

**Pull request merged**